test/sourceMap-option.test.js

```diff
@@ -137,54 +137,58 @@ describe('sourceMap', () => {
     expect(getWarnings(stats)).toMatchSnapshot('warnings');
   });
 
-  it('should match snapshot for a "true" value (the "devtool" option has the "source-map" value) and source map invalid', async () => {
-    const emitBrokenSourceMapPlugin = new (class EmitBrokenSourceMapPlugin {
-      apply(pluginCompiler) {
-        pluginCompiler.hooks.compilation.tap(
-          { name: this.constructor.name },
-          (compilation) => {
-            compilation.hooks.additionalChunkAssets.tap(
-              { name: this.constructor.name },
-              () => {
-                compilation.additionalChunkAssets.push('broken-source-map.js');
-
-                const assetContent = 'var test = 1;';
-
-                // eslint-disable-next-line no-param-reassign
-                compilation.assets['broken-source-map.js'] = {
-                  size() {
-                    return assetContent.length;
-                  },
-                  source() {
-                    return assetContent;
-                  },
-                  sourceAndMap() {
-                    return {
-                      source: this.source(),
-                      map: {},
-                    };
-                  },
-                };
-              }
-            );
-          }
-        );
-      }
-    })();
-    const compiler = getCompiler({
-      entry: `${__dirname}/fixtures/entry.js`,
-      devtool: 'source-map',
-      plugins: [emitBrokenSourceMapPlugin],
-    });
+  if (getCompiler.isWebpack4()) {
```

Comment:
Webpack@5 already throw error on broken source maps, right?

Reply:
source-map package does it

Reply:
:+1: